src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ForwardCancellationTokenToInvocationsTests.cs

```diff
@@ -3988,31 +3994,22 @@ End Class
             return VerifyVB.VerifyCodeFixAsync(originalCode, fixedCode);
         }
 
-        [Theory]
-        [InlineData("c As CancellationToken", "", "")]
-        [InlineData("", "(Of CancellationToken)", "c")]
-        public Task VB_Diagnostic_MultiNesting(string topMethodParam, string extensionTypeParam, string extensionArg)
+        [Fact]
+        public Task VB_Diagnostic_MultiNesting_TopMethod()
```

Comment:
Local methods are not available in VB, but the "MultiNesting" method was still relevant because lock block bodies exist there, so we need to look for the parent method ct.